### PR TITLE
White board dash

### PR DESCRIPTION
**Pull request by Manik-Coder**
Used Material UI elements to enhance the earlier version of Professors view on Courses

### CODE REVIEW

File: src/components/Login.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    setLoading,
+    user,
+    setUser
+  } = props;
+
+  return (
+    <section className="login">
+      <div className="loginContainer">
+        <h1 className="head" align="center">
+          Welcome{" "}
```
Comment (n-grounds) [inline src/components/Login.js]: what's the purpose of the `{"  "}`?

**Review by n-grounds — APPROVED** [review]
Seems like a lot of changes, even to things like courseServices, for a ticket presumably about the Dashboard.

File: src/components/Login.js (hunk 1 above)
Comment (Manik-Coder) [reply src/components/Login.js]: That would not effect the Functioning of the page .The presence of {" "} is for future update on the Welcome message if needed.

**Comment by logan-giese** [comment]
It seems like the large amount of conflicts might have been caused by manually copying changes from dev to "integrate" it, when a merge from "dev" to "WhiteBoardDash" would have updated changes properly.  If that's what happened, I recommend that to avoid problems in the future you should do a merge from dev to your branch.  From command line, just make sure you're on your branch and run "git merge dev" to merge dev into your branch.

File: src/components/Assigform.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import React ,{useState, useEffect} from 'react'
+
+
+const Courseform =(props)=> {
```
Comment (logan-giese) [inline src/components/Assigform.js:4]: Not sure why this is called "Courseform" when the file is called "Assigform.js"

**Review by logan-giese — APPROVED** [review]
Took some work to resolve everything, but it's finally ready for merge.

**Pull request merged**